pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -748,10 +746,7 @@ func isBlockMigration(vmi *v1.VirtualMachineInstance) bool {
 func (l *LibvirtDomainManager) generateMigrationProxies(vmi *v1.VirtualMachineInstance) []migrationproxy.MigrationProxyListener {
 	// create local migration proxies.
 	//
-	// Right now Libvirt won't let us perform a migration using a unix socket, so
```

Comment:
I'm just curious - what have changed that now libvirt lets us perform a migration using a unix socket?

Reply:
This RFE got implemented: https://bugzilla.redhat.com/show_bug.cgi?id=1638889